.github/workflows/early-access.yaml

```diff
@@ -52,12 +57,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-22.04, macOS-11, windows-2019 ]
+        os: [ ubuntu, macos, windows ]
```

Comment:
There are not runners like `ubuntu`
There are `ubuntu-latest` and so on ...

https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners